src/walletSdk/horizon/AccountService.ts

```diff
@@ -0,0 +1,18 @@
+import { Keypair } from "stellar-sdk";
+
+import { SigningKeypair } from "./Account";
+
+// Do not create this object directly, use the Wallet class.
+export class AccountService {
+  private server;
+  private network;
+
+  constructor(cfg) {
+    this.server = cfg.stellar.server;
+    this.network = cfg.stellar.network;
+  }
+
+  createKeypair() {
```

Comment:
could we add return type here?

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/21/commits/387e2f988d5d91f79734e0f27eceaf8662427fb2